app/models/address.rb

```diff
@@ -12,7 +12,7 @@ class Address < ApplicationRecord
 
   has_many :events
   has_many :partners
-  has_many :calendars
+  # has_many :calendars
```

Comment:
how come we are removing this relation? and if we are removing it should we delete rather comment, do we need to do any migration related tasks?